apps/mobile/src/utils/isFeatureEnabled.tsx

```diff
@@ -25,6 +25,9 @@ const ROLE_FLAGS: Record<Role, Record<FeatureFlag, boolean>> = {
   EARLY_ACCESS: {
     KOALA: true,
   },
+  EMAIL_TESTER: {
+    KOALA: true,
+  },
```

Comment:
hmm i think this should be:

```
ADMIN: {
  KOALA: true,
  EMAIL_TESTER: true,
},
BETA_TESTER: {
  KOALA: true,
  EMAIL_TESTER: false,
}
...
```

instead of a new field.

also, i think we can delete `KOALA` key now!

Reply:
@Robinnnnn  This one a little bit weird. I think we have `EMAIL_TESTER` in our graphql schema. Seems like we need to defined the `EMAIL_TESTER` key here

![CleanShot 2024-02-16 at 13 20 47@2x](https://github.com/gallery-so/gallery/assets/4480258/cbfe70cb-99e3-4863-af5d-4a022a8452dd)


Removed the `KOALA`!

Reply:
ohhh great catch!